isis/src/control/apps/findfeatures/findfeatures.cpp

```diff
@@ -88,12 +90,69 @@ namespace Isis{
       return;
   }
 
+  /**
+   * @brief Loads train images and applies geometry if provided
+   *
+   * This function loads a trainer image and optionally applies geometry using
+   * a FastGeom transform.
+   *
+   * A MatchImage is added to the MatchMaker as a trainer image. YOU SHOULD NOT
+   * USE THIS FUNCTION TO SET THE QUERY IMAGE. Once loaded, a FastGeom is
+   * optionally applied to the image before adding it to the matcher.
+   *
+   * Errors are trapped here if the file cannot be loaded or the geometry proess
+   * in FastGeom fails. In this case, false is returned without adding the image
+   * to the matcher.
+   *
+   * @author 2021-10-29 Kris J. Becker
+   *
+   * @param matcher    The source reponsible for managing the matcher process
+   * @param trainfile  File name to load
+   * @param fastgeom   Apply FastGeom if provded. Errors will return false with
+   *                      no adding to the matcher
```

Comment:
Looks like a typo, should be "return false without adding to the matcher"

Reply:
Fixed.